<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/Icon/Speed.ico</normaloff>:/Icon/Speed.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>button</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTest">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>65</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdTest">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>115</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_btnStatus">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>15</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Button Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPath">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbTest">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Model 1</string>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Icon/1.ico</normaloff>:/Icon/1.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Model 2</string>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Icon/2.ico</normaloff>:/Icon/2.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Model 3</string>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Icon/3.ico</normaloff>:/Icon/3.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Model 4</string>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Icon/4.ico</normaloff>:/Icon/4.ico</iconset>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="sldTest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabAuto">
     <attribute name="title">
      <string>Auto</string>
     </attribute>
     <widget class="QLabel" name="lbAuto">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabManual">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="lbManual">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>25</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbImg">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>285</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_checkboxStaus">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>60</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Check box status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_radioStatus">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>110</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Radio status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tabVal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Tab Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sldVal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>175</y>
      <width>46</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Radio status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cbbStatus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>15</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(170, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Combobox Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>250</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTest">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
